.github/pull_request_template.md

```diff
@@ -6,7 +6,7 @@
 
 **PR Checklist:**
 
-- [ ] Code is formatted (run `pre-commit run --all-files`)
+- [ ] Code is formatted (`pre-commit` is installed)
```

Comment:
```suggestion
- [ ] `pre-commit` hooks pass locally
```

The **pre-commit** hooks aren't just formatting, so maybe this text is more correct? What do you think?